### PR TITLE
Add "Audio/Video Updates in Chrome 69" /updates article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
**Target Live Date:** Same as Picture-in-Picture article

- [x] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

Live preview at https://pr-6435-dot-web-central.appspot.com/web/updates/2018/08/chrome-69-media-updates

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@jpmedley Can you have a look please?

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+need to enable the experimental "Web Platform Features" flag at
+`chrome://flags/#enable-experimental-web-platform-features`.
+
+[Intent to Experiment](https://www.google.com/url?q=https://groups.google.com/a/chromium.org/forum/%23!topic/blink-dev/ITzZ_yx4bF8&sa=D&ust=1530528558634000&usg=AFQjCNFc5O4gFlE5rBSytmc73mGgqCyNgA) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5652917147140096) &#124;
+[Chromium Bug](https://crbug.com/709348)
+
+## AV1 decoder {: #av1 }
+
+TODO
```
Comment (kaycebasques) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: TODO

**Comment by beaufortfrancois** [comment]
@kaycebasques AV1 section has been added.

**Comment by beaufortfrancois** [comment]
**⚠** If article looks good, please merge and push live just **before** Chrome 69 Beta Blog Post is published.

**Comment by PaulKinlan** [comment]
Is there a hard constraint about timing? I could pull it in now because I think we are pretty close to the post going out.

**Comment by jpmedley** [comment]
Looking now. Sorry for the delay.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        // TODO: Fetch high resolution protected content...
+      });
+    })
+    .catch(error => {
+      // TODO: Fallback to fetch license or stream low-resolution content...
+    });
+
+### Available for Origin Trials
+
+In order to get feedback from web developers, the HDCP Policy Check API is
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: Delete 'In order'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+available as an [Origin Trial] in Chrome 69 for Desktop (Chrome OS, Linux, Mac,
+and Windows). You will need to [request a token], so that the feature would be
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: 'would be' > 'is'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+automatically enabled for your origin for a limited period of time, without the
+need to enable the experimental "Web Platform Features" flag at
+`chrome://flags/#enable-experimental-web-platform-features`.
+
+[Intent to Experiment](https://www.google.com/url?q=https://groups.google.com/a/chromium.org/forum/%23!topic/blink-dev/ITzZ_yx4bF8&sa=D&ust=1530528558634000&usg=AFQjCNFc5O4gFlE5rBSytmc73mGgqCyNgA) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5652917147140096) &#124;
+[Chromium Bug](https://crbug.com/709348)
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: See my comment above.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+
+## MSE PTS/DTS compliance {: #pts }
+
+Buffered ranges and duration values are now reported by Presentation Time Stamp
+(PTS) intervals, rather than by Decode Time Stamp (DTS) intervals in [Media
+Source Extensions (MSE)].
+
+At the very beginnings of MSE, Chrome’s implementation was tested against WebM
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: 'At the very beginnings of MSE' > 'When MSE was new'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+and MP3, some media stream formats where there was no such distinction between
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: Delete 'such'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+PTS and DTS. And it was working fine… until ISO BMFF (aka MP4) was added. This
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: Delete the ellipsis.
'This' > 'This container'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+frequently contains out-of-order presentation versus decode time streams (eg,
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: I don't know what you mean by 'out-of-order presentation versus decode time streams'.

The abbreviation e.g. should contain periods. Pro tip: e.g. and i.e. are easy to mix up. For the of clarity I always use 'for example' and 'in other words'.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 9]
```diff
@@ -0,0 +1,311 @@
...
+for codecs like H.264) causing DTS and PTS to differ. That caused Chrome to
+report (usually just slightly) different buffered ranges and duration values
+than expected. But here it is: this new behavior will roll out gradually in
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: Delete 'But here it is'. It's not adding anything.

What do you mean roll out gradually in 69? It's either there or it isn't.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 10]
```diff
@@ -0,0 +1,311 @@
...
+Chrome 69 and makes its MSE implementation compliant with the [MSE
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: 'makes' > 'make'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 11]
```diff
@@ -0,0 +1,311 @@
...
+  </figcaption>
+</figure>
+
+This change affects `MediaSource.duration` (and consequently
+`HTMLMediaElement.duration`), `SourceBuffer.buffered` (and consequently
+`HTMLMediaElement.buffered)`, and `SourceBuffer.remove(start, end)`.
+
+If you’re not sure which method is used to report buffered ranges and duration
+values, you can go to the internal `chrome://media-internals` page and search for
+"ChunkDemuxer: buffering by PTS" or  "ChunkDemuxer: buffering by DTS" in logs.
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: 'in logs' > 'in the logs'

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 12]
```diff
@@ -0,0 +1,311 @@
...
+
+## Handling of media view intents on Android Go {: #media_intent_handler }
+
+[Android Go] is a lightweight version of Android designed for entry-level
+smartphones. To that end, it does not necessarily ship with some media-viewing
+applications, so if a user tries to open a downloaded video for instance, they
+won’t have any applications to handle that intent.
+
+To fix this, Chrome 69 on Android Go now listens for media-viewing intents so
+users can view downloaded audio, videos and images.
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: Comma after videos.

Are you saying Chrome takes the place of the missing viewing applications? That point could be clearer.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 13]
```diff
@@ -0,0 +1,311 @@
...
+`SourceBuffer.appendBuffer()` in the last 3 seconds.
+
+As websites may decide to append large chunks of data at a low frequency, this
+is not a useful signal about buffering health. Removing "stalled" events for
+media elements using MSE clears up confusion and brings Chrome more in line
+with the MSE specification. Note that media elements that don't use MSE will
+continue to raise "stalled" events as they do today.
+
+[Intent to Deprate and Remove](https://groups.google.com/a/chromium.org/d/msg/blink-dev/x54XtrTyOP8/4-5QZlZzDAAJ) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=836951)
```
Comment (jpmedley) [inline src/content/en/updates/2018/07/chrome-69-media-updates.md]: Here's the Chrome status entry. Make sure these are in the same order as elsewhere.

https://www.chromestatus.com/features/6338037575319552

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 14]
```diff
@@ -0,0 +1,311 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the audio/video updates in Chrome 69.
+
+{# wf_updated_on: 2018-07-31 #}
+{# wf_published_on: 2018-07-27 #}
+{# wf_tags: news,chrome69,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: AV1, Picture-in-Picture, and HDCP policy check are here! #}
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 15]
```diff
@@ -0,0 +1,311 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the audio/video updates in Chrome 69.
+
+{# wf_updated_on: 2018-07-31 #}
+{# wf_published_on: 2018-07-27 #}
+{# wf_tags: news,chrome69,media #}
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 16]
```diff
@@ -0,0 +1,311 @@
...
+- Android Go users can [open downloaded audio, video and images in Chrome](#media_intent_handler).
+- [Stalled events](#stalled) for media elements using MSE are removed.
+
+## AV1 video decoder {: #av1 }
+
+AV1 is a next generation codec developed by the [Alliance for Open Media] which
+[improves compression efficiency by 30%] over the current state-of-the-art
+video codec, VP9.
+
+Chrome 69 adds an AV1 decoder into Chrome Desktop (Windows, Mac, Linux,
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 17]
```diff
@@ -0,0 +1,311 @@
...
+supported container is [ISO-BMFF (MP4)]. See [From raw video to web ready] for
+a brief explanation of containers.
+
+[Chromestatus Tracker](https://www.chromestatus.com/features/5729898442260480)
+
+## Watch video using Picture-in-Picture {: #picture_in_picture }
+
+Picture-in-Picture (PiP) allows users to watch videos in a floating window
+(always on top of other windows) so they can keep an eye on what they’re
+watching while interacting with other sites, or applications. With the new
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 18]
```diff
@@ -0,0 +1,311 @@
...
+[Picture-in-Picture Web API], you can initiate and control Picture-in-Picture
+for video elements on your website. Read our [article] to learn all about it.
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 19]
```diff
@@ -0,0 +1,311 @@
...
+description: A round up of the audio/video updates in Chrome 69.
+
+{# wf_updated_on: 2018-07-31 #}
+{# wf_published_on: 2018-07-27 #}
+{# wf_tags: news,chrome69,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: AV1, Picture-in-Picture, and HDCP policy check are here! #}
+{# wf_blink_components: Blink>Media #}
+
+# Audio/Video Updates in Chrome 69 {: .page-title }
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 20]
```diff
@@ -0,0 +1,311 @@
...
+       alt="Picture-in-Picture window">
+  <figcaption>
+    <b>Figure 1.</b>
+    Picture-in-Picture window
+  </figcaption>
+</figure>
+
+## EME: Querying Encryption Scheme Support {: #encryption_scheme }
+
+Some platforms or key systems only support CENC mode, while others only support
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 21]
```diff
@@ -0,0 +1,311 @@
...
+CBCS mode. Still others are able to support both. These two encryption schemes
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 22]
```diff
@@ -0,0 +1,311 @@
...
+are incompatible, so web developers must be able to make intelligent choices
+about what content to serve.
+
+To avoid having to determine which platform they’re on to check for “known”
+encryption schemes support, a new `encryptionScheme` key is [added] in
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 23]
```diff
@@ -0,0 +1,311 @@
...
+`MediaKeySystemMediaCapability` to allow websites to specify which encryption
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: I've used https://w3c.github.io/encrypted-media/#idl-def-mediakeysystemmediacapability which is the correct one.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 24]
```diff
@@ -0,0 +1,311 @@
...
+schemes could be used in [Encrypted Media Extensions (EME)].
+
+The new `encryptionScheme` key can be of two values:
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 25]
```diff
@@ -0,0 +1,311 @@
...
+
+- `'cenc'` AES-CTR mode full sample and video NAL subsample encryption.
+- `'cbcs'` AES-CBC mode partial video NAL pattern encryption.
+
+If not specified, it indicates that any encryption scheme is acceptable. Note
+that [Clear Key] always supports the `'cenc'` scheme.
+
+Snippet below shows you how to query two configurations with different
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md [hunk 26]
```diff
@@ -0,0 +1,311 @@
...
+            contentType: 'audio/mp4; codecs="mp4a.40.2"',
+            encryptionScheme: 'cbcs'
+          },
+        ],
+        initDataTypes: ['keyids']
+      }]);
+
+[Intent to Implement](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/lMUKOaohUTY) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5184416120832000) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=838416)
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: I'll make sure it is added everywhere else.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md (hunk 8 above)
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.

File: src/content/en/updates/2018/07/chrome-69-media-updates.md (hunk 9 above)
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: See https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/I6oGJLym-Tk/Z46le9l4CQAJ

File: src/content/en/updates/2018/07/chrome-69-media-updates.md (hunk 10 above)
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done

File: src/content/en/updates/2018/07/chrome-69-media-updates.md (hunk 11 above)
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done

File: src/content/en/updates/2018/07/chrome-69-media-updates.md (hunk 12 above)
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done.
Thanks!

File: src/content/en/updates/2018/07/chrome-69-media-updates.md (hunk 13 above)
Comment (beaufortfrancois) [reply src/content/en/updates/2018/07/chrome-69-media-updates.md]: Done

**Comment by beaufortfrancois** [comment]
@jpmedley I've addressed all your feedback. I'll ask for merge shortly.

File: src/content/en/updates/2018/08/chrome-69-media-updates.md [hunk 27]
```diff
@@ -0,0 +1,321 @@
...
+  </figcaption>
+</figure>
+
+Note that this Chrome feature is enabled on all Android devices running Android
+O and onwards with 1 GB of RAM or less.
+
+[Chromium Bug](https://crbug.com/718641)
+
+## Removal of “stalled” events for media elements using MSE {: #stalled }
+
```
Comment (jpmedley) [inline src/content/en/updates/2018/08/chrome-69-media-updates.md]: Francois,

Can you do me a favor can you add this macro right after the heading (line 272):

{% include "web/updates/_shared/see-all-dep-rem.html" %}

You can see what it looks like [here](https://developers.google.com/web/updates/2018/06/chrome-68-deps-rems).

**Comment by beaufortfrancois** [comment]
Closing as the other one was pushed.

**Pull request closed without merging**